Makefile

```diff
@@ -6,6 +6,8 @@ UI_BUILD_PATH := ${UI_PATH}/build
 API_PATH := api
 API_ALL_PACKAGES := $(shell cd ${API_PATH} && go list ./... | grep -v github.com/gojek/mlp/client | grep -v mocks)
 BIN_NAME := $(if ${APP_NAME},${APP_NAME},mlp)
+DOCKER_COMPOSE_BIN := "docker-compose"
```

Comment:
Yes, to make it easier to use `nerdctl` instead of `docker-compose` (i.e. a drop-in replacement for `docker-compose` command is `nerdctl compose`). Unfortunately, `Make` doesn't recognize shell aliases, so this is a "not so nice" way of making it possible to use `nerdctl` instead. 
An alternative to this would be to create an executable shell script called `docker-compose` somewhere under `PATH`, which will be just running `nerdctl compose -- $@` instead.